trin-history/src/storage.rs

```diff
@@ -642,11 +641,11 @@ pub mod test {
             let value: Vec<u8> = vec![0; 32000];
```

Comment:
Can we add a check for `network_content_storage_used` attribute here? Just check that if in each iteration, it is the expected size. If we can also add a check inside a test to make sure that it's decremented appropriately, that would be great